x/claim/client/cli/cli_test.go

```diff
@@ -105,71 +106,109 @@ func (s *IntegrationTestSuite) TearDownSuite() {
 	s.network.Cleanup()
 }
 
-func (s *IntegrationTestSuite) TestCmdQueryClaimable() {
+// TODO: Make sure CLI replaces with genesis time of chain.
+// Other than that, this works
+
+// func (s *IntegrationTestSuite) TestGetCmdQueryParams() {
```

Comment:
Why is this commented out?

Reply:
Because of the comment above:

Because the airdrop_start_time is the genesis time of the test chain, but I don't know how to get that here, as I couldn't find it in s.network.Config.